conans/util/thread.py

```diff
@@ -15,3 +20,29 @@ def join(self, timeout=None):
     def raise_errors(self):
         if self._exc:
             raise self._exc
+
+
+def cpu_count():
```

Comment:
Move it back to its original location if at the end it is not a necessary change for this PR?